modules/split_wav.js

```diff
@@ -0,0 +1,202 @@
+//Example usages
+// const vttToJson = require('vtt-json');
+// const fs = require('fs');
+// const {splitWavParallel, splitWavSequential} = require('./split_wav_by_json');
+
+// (async() => {
+//   var vtt = fs.readFileSync('sample.vtt').toString();
+//   var json = await vttToJson(vtt);
+//   // console.log(json);
+//   var ret = await splitWavParallel('sample.wav', json);
+//   console.log(ret);
+// })();
+
+const {spawn} = require('child_process');
```

Comment:
Could you use the promisified version of this library?

Reply:
Using the promisified version gives over my control of error logging.

Reply:
https://www.npmjs.com/package/child-process-promise
Could you elaborate further on what logging will this library prevent you from doing?

Reply:
There are two ways the process throws an error, one is the process was not created correctly, and one is the process exits non-zero. I will not be able to deal with the case where the process exits non-zero.

Reply:
```
const { spawn } = require('child-process-promise');
        const ffmpeg = spawn('ffmpeg', ['-nostdin', '-i', pathToFile, '-c:a', 'pcm_s16le', '-ac', '1', '-y', '-ar', '16000', outputFile]);
        var res = await ffmpeg;
        console.log(res.childProcess.exitCode);
```
If your issue is with accessing the exitCode, you could do as specified above, 
Try doing a `console.log(res.childProcess)` there are more details available in that object